d3a_api_client/commands.py

```diff
@@ -14,13 +14,13 @@ def offer_energy(self, area_uuid, energy, price):
         return self._add_to_buffer(area_uuid, Commands.OFFER, {"energy": energy, "price": price})
 
     def offer_energy_rate(self, area_uuid, energy, rate):
-        return self._add_to_buffer(area_uuid, Commands.OFFER, {"energy": energy, "rate": rate})
+        return self._add_to_buffer(area_uuid, Commands.OFFER, {"energy": energy, "price": rate})
```

Comment:
Side bug fix

Reply:
You need to multiply rate * energy to calculate the correct price value.

Reply:
@spyrostz It depends on whether the user will want to pass rate here or the price , what do you think would make more sense ?

Reply:
Given the name of the method, I actually think the user would likely pass the rate

Reply:
@fievelk I agree, @spyrostz what do you think?

Reply:
The method is named `offer_energy_rate` , therefore the user needs to pass the energy rate in cents / kWh here. The api parameter is named price, therefore the API requires the price in cents. Therefore the client has to act as a translator and convert the rate -> price. At least that's my understanding, also have a look at the offer_energy_rate in the rest / redis devices, you will see the conversion.